app.js

```diff
@@ -0,0 +1,98 @@
+var app = angular.module('PathOfDamage', []);
+app.controller('Damage', function ($scope) {
+  $scope.hits = [100, 500, 1000, 2000, 3000, 4000, 5000, 7500, 10000];
+
+  $scope.buffs = {
+    total: 0,
+    values: []
+  };
+  $scope.shifts = {
+    total: 0,
+    values: []
+  };
+  $scope.damageReductions = {
+    total: 0,
+    values: []
+  };
+  $scope.flatTaken = {
+    total: 0,
+    values: []
+  };
+  $scope.reducedTaken = {
+    total: 0,
+    values: []
+  };
+  $scope.lessTaken = {
+    total: 0,
+    values: []
+  };
```

Comment:
You could make an init method that takes an array of these modifiers (or just takes a variable number of arguments) and for each of them initializes it to {total:0, values: []}

its kind of nice to do it the way you have it, though, since you can document each one in comments as it is initialized.